data/json/items/comestibles/raw_fruit.json

```diff
@@ -472,279 +472,352 @@
+    "volume": "250 ml",
     "fun": 4,
     "flags": [ "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 93 ], [ "calcium", 2 ], [ "iron", 3 ] ]
+    "vitamins": [ [ "vitC", 87 ], [ "calcium", 1 ], [ "iron", 2 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "coconut",
     "name": { "str": "coconut" },
-    "weight": "84 g",
+    "weight": "680 g",
     "color": "brown",
     "spoils_in": "360 days",
-    "comestible_type": "FOOD",
+    "comestible_type": "INVALID",
     "symbol": "%",
-    "quench": 2,
-    "calories": 299,
+    "quench": 10,
+    "calories": 1940,
     "description": "A tropical fruit with a hard, hairy shell and a hollow inside full of liquid.",
     "price": 100,
     "price_postapoc": 100,
     "material": [ "fruit" ],
-    "volume": "2 L",
+    "volume": "1850 ml",
     "bashing": 2,
-    "charges": 5,
-    "vitamins": [ [ "vitC", 3 ], [ "calcium", 1 ], [ "iron", 14 ] ],
-    "fun": 1
+    "vitamins": [ [ "vitC", 15 ], [ "calcium", 4 ], [ "iron", 36 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "coconut_flesh",
+    "name": { "str": "handful of coconut flesh", "str_pl": "handfuls of coconut flesh" },
+    "weight": "80 g",
+    "color": "brown",
+    "spoils_in": "3 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 5,
+    "calories": 82,
+    "description": "A handful of coconut flesh.  Quite shockingly caloric.",
+    "price": 283,
+    "price_postapoc": 50,
+    "material": [ "fruit" ],
+    "volume": "250 ml",
+    "fun": 4,
+    "flags": [ "SMOKABLE" ],
+    "smoking_result": "dry_fruit",
+    "vitamins": [ [ "vitC", 2 ], [ "iron", 10 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "peach",
     "name": { "str": "peach", "str_pl": "peaches" },
-    "weight": "163 g",
+    "weight": "150 g",
     "color": "yellow",
     "spoils_in": "3 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 4,
-    "calories": 63,
+    "calories": 58,
     "description": "This fruit's large pit is surrounded by its tasty flesh.",
-    "price": 140,
+    "price": 38,
     "price_postapoc": 100,
     "material": [ "fruit" ],
-    "volume": "250 ml",
+    "volume": "180 ml",
     "fun": 4,
     "flags": [ "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 12 ], [ "calcium", 1 ], [ "iron", 2 ] ]
+    "vitamins": [ [ "vitC", 11 ], [ "iron", 2 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "watermelon",
     "name": { "str": "watermelon" },
-    "weight": "161 g",
+    "weight": "4518 g",
     "color": "green",
     "spoils_in": "7 days 12 hours",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "quench": 28,
-    "//": "yes, it really does have that much water in it",
-    "calories": 48,
+    "quench": 826,
+    "//": "yes, it really does have that much water in it - 4131.7g in this size specifically",
+    "calories": 1355,
     "description": "A deep green melon, bigger than your head.  It is very juicy!",
```

Comment:
A deep green melon, bigger than your head.  It is very juicy!  You should find something to slice it up, as any other method of cracking open and eating it would be very messy.